### ISSUE TITLE
The App seems to be slow !!

### ISSUE THREAD

**Issue by DhairyaBahl** [DhairyaBahl/React-Messenger-App#95]
### Description
App is working very slowly. It would be great if it can be a bit faster.

@abhijeet007rocks8 I have deployed all the changes ! Kindly confirm if it is slow only at my end or you also feel it is laggy ! Kindly lead this issue.

**Comment by abhijeet007rocks8** [comment]
it just takes a little bit more time due to sheer volume of messages

**Comment by abhijeet007rocks8** [comment]
Still ill take a look if i can make it faster somehow

**Comment by DhairyaBahl** [comment]
@abhijeet007rocks8 Thank You 😄

**Comment by AnjaliPatle** [comment]
@abhijeet007rocks8 I think we don't have a loading element in this app right now. I think we can put a simple loading element while the messages load to make the wait-time less frustrating for users. If you would like to include this feature then please let me know as I'd like to work on it by creating an issue for it!

**Comment by abhijeet007rocks8** [comment]
@AnjaliPatle Nice idea go ahead and make the issue

**Comment by Jaagrav** [comment]
I can fix this, we can simply implement React-Virtualized lists in order to make the app load faster. Due to an enormous amount of messages, the first paint of the app takes quite long to load. You can look into the docs of [React-Virtualized](https://bvaughn.github.io/react-virtualized/)

**Comment by DhairyaBahl** [comment]
@Jaagrav I think someone is already assigned to this issue.

@abhijeet007rocks8 If no-one is working on this issue right now, then kindly assign him.

**Comment by abhijeet007rocks8** [comment]
@Jaagrav other participant is working on lading component you can work on improving loading time of the app.

**Comment by abhijeet007rocks8** [comment]
We can see about lazy loading in which the content is loaded when you scroll to that part.
Refer This https://www.geeksforgeeks.org/what-is-lazy-loading/

**Comment by akshay1027** [comment]
Why not switch to next js ? That would be boost performance like crazy as it.uses SRR @abhijeet007rocks8 @DhairyaBahl . But it would be quite hectic to change I guess

**Comment by abhijeet007rocks8** [comment]
> Why not switch to next js ? That would be boost performance like crazy as it.uses SRR @abhijeet007rocks8 @DhairyaBahl . But it would be quite hectic to change I guess

Yes I surely agree, I personally work using nextJS and styled-components/Scss its superfast but migrating entire project to NextJS is like making a completely new project.

**Comment by akshay1027** [comment]
Agreed 🙌🏻

**Comment by subhangi2731** [comment]
@abhijeet007rocks8 please assign me this issue
